specification/_types/aggregations/bucket.ts

```diff
@@ -671,21 +671,36 @@ export class RangeAggregation extends BucketAggregationBase {
   format?: string
 }
 
-export class AggregationRange {
+/**
+ * @codegen_names untyped, number, string
+ * @variants untagged untyped=_types.aggregations.UntypedAggregationRange
+ */
+export type AggregationRange =
+  | UntypedAggregationRange
+  | NumberAggregationRange
+  | StringAggregationRange
```

Comment:
Let's check, if we can have "term" range aggregations and if yes, rename this variant to TermAggregationRange.

We as well most likely want to have either a Date- or even a DateMath variant.

Reply:
[DateRangeAggregation](https://github.com/elastic/elasticsearch-specification/blob/5615a46a870f15fa51edf6a3c4e8c2ed8cc3ebf7/specification/_types/aggregations/bucket.ts#L307) can be part of the union yes. so untyped | number | term | date,  like range query?

Reply:
just noticed that DateRangeAggregation, like IpRangeAggregation, has its own field in the AggregationContainer (`date_range`). does it make sense to also add it here?